test/groovy/NeoDeployTest.groovy

```diff
@@ -417,6 +423,24 @@ class NeoDeployTest extends BasePiperTest {
 
     }
 
+    @Test
+    void invalidateCache_Success_Test() {
+
+        nullScript.commonPipelineEnvironment.configuration = [steps: [neoDeploy: [neo: [host: 'test.deploy.host.com', account: 'trialuser123', invalidateCache: true, oauthCredentialId: 'testOauthId', siteId: "12346"]]]]
+        fileExistsRule.registerExistingFile('./target/artifact.war')
+
+        shellRule.setReturnValue(JenkinsShellCallRule.Type.REGEX, "https:\\/\\/oauthasservices-trialuser123\\.test\\.deploy\\.host\\.com\\/oauth2\\/api\\/v1\\/token", "{\"access_token\":\"xxx\"}")
+        shellRule.setReturnValue(JenkinsShellCallRule.Type.REGEX, "fiori\\/api\\/v1\\/csrf", "{\"X-CSRF-Token\":\"xxx\"}")
```

Comment:
In reality this does not return the xcsrf token as a json, but as a header. That is why readProperties is used in the productive code. The test should IMO reflect that.

Reply:
done